dev_tools/requirements/envs/dev.env.txt

```diff
@@ -2,18 +2,14 @@
 # This file is autogenerated by pip-compile with Python 3.10
 # by the following command:
 #
-#    pip-compile --output-file=dev.env.txt --resolver=backtracking deps/dev-tools.txt deps/runtime.txt
+#    pip-compile --output-file=envs/dev.env.txt deps/dev-tools.txt deps/runtime.txt
```

Comment:
Does pip-tools allow you to assert hashes?

Reply:
it can generate hashes